template/index.js

```diff
@@ -8,8 +8,18 @@ const flexboxgrid = fs.readFileSync(path.join(__dirname, './deps/flexboxgrid.min
 const chartistStyle = fs.readFileSync(path.join(__dirname, './deps/chartist.min.css'))
 const chartistScript = fs.readFileSync(path.join(__dirname, './deps/chartist.min.js'))
 
-module.exports = function (results) {
-  const bodyTree = report(results)
+module.exports = function (results, compare) {
+  if (compare && compare.length > 0) {
+    compare.forEach(function (val) {
+      if (val.start === results.start && val.finish === results.finish) {
+        var index = compare.indexOf(val)
+        if (index > -1) {
+          compare.splice(index, 1)
+        }
+      }
+    })
```

Comment:
You should never remove indexes from an array when iterating through them from the beginning, because that means that when you remove something and move to the next index, something undesirable could slip into the index you just checked. Its a super common gotcha. Start from the end of the array and go backwards to avoid this. :) The second param of the forEach function is the index, or using a for loop will give you the index, so you don't need to search for the index again (it adds more complexity you wouldn't need)